boards/nxp/frdm_ke17z/frdm_ke17z.dts

```diff
@@ -60,6 +60,15 @@
 	};
 };
 
+&idle {
+	min-residency-us = <1>;
+};
+
+&stop {
+	min-residency-us = <20000>;
+	exit-latency-us = <13>;
+};
```

Comment:
do we not need this information for pstop1 & pstop2?

Reply:
No, partial Stop is a clocking option that can be taken instead of entering Stop mode and is configured in the SMC Stop Control Register (SMC_STOPCTRL).